python/heterocl/schedule.py

```diff
@@ -292,66 +291,327 @@ def to(self, tensors, dst, src=None, axis=0,
             Move axis-th loop body to xcel scope
 
         mode : data movement type
-            The modes of data movement (FIFO, DMA, MMIO)
-            For inter-kernel data movemnet, only FIFO is supported
+            The modes of data movement (Stream, DMA, MMIO)
+            For inter-kernel data movemnet, only Stream is supported
 
         depth : channel depth
```

Comment:
The RHS after `:` should be the type. In this case, it should be
```suggestion
        depth : int
```

Reply:
Please also update other args in the comment.